frontend/tsconfig.json

```diff
@@ -22,5 +22,5 @@
   },
   "include": [
     "src"
-  ]
+, "learningCode/test", "learningCode/Test.state.ts", "learningCode/Test.reducer.ts", "learningCode/Test.actions.tsx"  ]
```

Comment:
```suggestion
```
We don't need to include test/learning code here, especially now that we changed those files to be .txt